UI/src/view/MassTransitStopLayer.js

```diff
@@ -354,14 +354,15 @@ window.MassTransitStopLayer = function(map, roadCollection, mapOverlay, assetGro
   };
 
   var createNewAsset = function(lonlat, placement) {
+    var directions_decode = {BothDirections: 2, TowardsDigitizing: 2, AgainstDigitizing: 3};
```

Comment:
Okay, let's rename the mapping then. How about renaming the variable as "defaultAssetDirection"?